src/cargo/core/profiles.rs

```diff
@@ -666,6 +669,7 @@ pub struct Profile {
     pub incremental: bool,
     pub panic: PanicStrategy,
     pub strip: Strip,
+    pub rustflags: Option<InternedString>,
```

Comment:
I think that this would be stored as `Vec<InternedString>` to avoid the join-with-space followed by a split-on-space.

Reply:
I was going to go with that approach but got an error about not being able to implement `Copy` for `Profile`.
I can change it to something else, but not sure what else would be preferred.

Reply:
I'm not sure how to best fix that myself, but we can't join/split on spaces because that breaks arguments with spaces in them (they would become multiple arguments instead of one)